addons/website/static/src/scss/website.scss

```diff
@@ -314,13 +340,54 @@ h5, h6 {
 h6 {
     color: map-get($colors, 'o-cc1-h6');
 }
+h1 {
+    line-height: $headings-line-height;
+    margin-top: $headings-margin-top;
+    margin-bottom: $headings-margin-bottom;
+}
+h2, .h2 {
+    font-family: $h2-font-family;
+    line-height: $h2-line-height;
+    margin-top: $h2-margin-top;
+    margin-bottom: $h2-margin-bottom;
+}
+h3, .h3 {
+    font-family: $h3-font-family;
+    line-height: $h3-line-height;
+    margin-top: $h3-margin-top;
+    margin-bottom: $h3-margin-bottom;
+}
+h4, .h4 {
+    font-family: $h4-font-family;
+    line-height: $h4-line-height;
+    margin-top: $h4-margin-top;
+    margin-bottom: $h4-margin-bottom;
+}
+h5, .h5 {
+    font-family: $h5-font-family;
+    line-height: $h5-line-height;
+    margin-top: $h5-margin-top;
+    margin-bottom: $h5-margin-bottom;
+}
+h6, .h6 {
+    font-family: $h6-font-family;
+    line-height: $h6-line-height;
+    margin-top: $h6-margin-top;
+    margin-bottom: $h6-margin-bottom;
+}
```

Comment:
Whatever we decide for what I said earlier (does h6 depends on h1 or on h5 which depends on h4, etc), it should in the end not be done like that. This always creates 6 CSS rules while most of the time, we'll have the same line height, font-family, etc, or at least many common rules.

For colors for example, we have this:
```scss
            // Headings
            h1, h2, h3, h4, h5, h6 {
                // 'inherit' comes from the o-bg-color mixin
                color: o-color($-headings);
            }
            h2, h3, h4, h5, h6 {
                color: o-color($-h2);
            }
            h3, h4, h5, h6 {
                color: o-color($-h3);
            }
            h4, h5, h6 {
                color: o-color($-h4);
            }
            h5, h6 {
                color: o-color($-h5);
            }
            h6 {
                color: o-color($-h6);
            }
```
=> Most of the time, only the first rule generates CSS.